src/Dotnet.Script/Program.cs

```diff
@@ -138,6 +138,8 @@ private static int Wain(string[] args)
                 var dllOption = c.Option("--dll", "Publish to a .dll instead of a .exe", CommandOptionType.NoValue);
                 var commandConfig = c.Option("-c | --configuration <configuration>", "Configuration to use for running the script [Release/Debug] Default is \"Debug\"", CommandOptionType.SingleValue);
                 var publishDebugMode = c.Option(DebugFlagShort + " | " + DebugFlagLong, "Enables debug output.", CommandOptionType.NoValue);
+                var runtime = c.Option("-r |--runtime", "The runtime to publish the self contained EXE to. Defaults to the your current platform.", CommandOptionType.SingleValue);
```

Comment:
Should maybe be "Defaults to your current runtime" instead of `platform`. So we don't mix up the terminology :)
Also maybe "The runtime to publish the self contained executable to" instead of "EXE" since that is sort of Windows only :)

Reply:
makes sense, I updated the texts